src/Helper/ValidateStreet.php

```diff
@@ -60,14 +60,15 @@ public static function validate(string $fullStreet, string $localCountry, ?strin
      */
     public static function getStreetRegexByCountry(string $local, string $destination): ?string
     {
-        if (
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL) ||
-            ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_NL)
-        ) {
+        $localIsNlOrBe = in_array($local, [AbstractConsignment::CC_BE, AbstractConsignment::CC_NL]);
+        $destinationIsNl = $destination === AbstractConsignment::CC_NL;
+        $destinationIsBe = $destination === AbstractConsignment::CC_BE;
+
+        if ($localIsNlOrBe && $destinationIsNl) {
             return self::SPLIT_STREET_REGEX_NL;
         }
 
-        if ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_BE) {
+        if ($localIsNlOrBe && $destinationIsBe) {
```

Comment:
Deze klopt niet helemaal. Van NL naar BE is het niet nodig om een regex te gebruiken.

Van NL naar NL:
NL regex

Van BE naar NL:
NL regex

Van BE naar BE
BE regex

De rest moet null returnen. Je kan het zelf controleren in de backoffice.